packages/components/dashboard/status-card/status-card.tsx

```diff
@@ -0,0 +1,82 @@
+import * as React from 'react';
+import { useK8sWatchResource } from '@openshift-console/dynamic-plugin-sdk';
+import { HealthBody } from '@openshift-console/dynamic-plugin-sdk-internal';
+import * as _ from 'lodash';
+import { useTranslation } from 'react-i18next';
+import {
+  Gallery,
+  GalleryItem,
+  Card,
+  CardBody,
+  CardHeader,
+  CardTitle,
+  Divider,
+  CardActions,
+  Dropdown,
+  KebabToggle,
+  DropdownItem,
+} from '@patternfly/react-core';
+import { MCG_OPERATOR } from '../../../constants';
+import { ClusterServiceVersionKind } from '../../../types';
+import HealthItem from '../../../utils/dashboard/status-card/HealthItem';
+import { operatorResource } from '../../resources';
+import { getOperatorHealthState } from '../utils';
+import './status-card.scss';
+
+const dropdownItems = [
+  <DropdownItem key="edit">Edit Subscription</DropdownItem>,
+  <DropdownItem key="remove">Remove Subscription</DropdownItem>,
+  <DropdownItem key="uninstall">Uninstall Operator</DropdownItem>,
+];
+
+export const StatusCard: React.FC = () => {
+  const [isOpen, setIsOpen] = React.useState<boolean>(false);
+  const { t } = useTranslation();
+  const [csvData, csvLoaded, csvLoadError] =
+    useK8sWatchResource<ClusterServiceVersionKind[]>(operatorResource);
```

Comment:
```suggestion
  const { t } = useTranslation();
  const [isOpen, setIsOpen] = React.useState<boolean>(false);
  const [csvData, csvLoaded, csvLoadError] =
    useK8sWatchResource<ClusterServiceVersionKind[]>(operatorResource);
```
Keep all hooks together